internal/common/common.go

```diff
@@ -104,3 +104,12 @@ func TarAwsFiles(ctx context.Context, tarName string, files, tarredFilenames []s
 	wg.Wait()
 	return err
 }
+
+func All(vs []string, f func(string) bool) bool {
```

Comment:
This is not generic and only works on strings, so the name `All` might not be the best here